receiver/sqlserverreceiver/scraper.go

```diff
@@ -508,6 +508,7 @@ func (s *sqlServerScraperHelper) recordDatabaseStatusMetrics(ctx context.Context
 func (s *sqlServerScraperHelper) recordDatabaseQueryTextAndPlan(ctx context.Context, topQueryCount uint) (pcommon.Resource, error) {
 	// Constants are the column names of the database status
 	const (
+		eventName      = "db.server.top_query"
```

Comment:
Are the constants coming from somewhere else? (Are they somewhere that we can reference as the source of the naming, or even import the constants?)

Reply:
The event names are not on the semantic conventions. This PR updates the event names to make them more formal and consistent.

The `user.name` attribute is documented here: [OpenTelemetry Semantic Conventions - User Attributes](https://opentelemetry.io/docs/specs/semconv/attributes-registry/user/)